.github/benchmark-files/run-phpstan.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+start_time_ms=$(date +%s%3N)
+
+vendor/bin/phpstan --memory-limit=-1 --debug --xdebug > /dev/null 2>&1
+
+end_time_ms=$(date +%s%3N)
+
+elapsed_time_ms=$((end_time_ms - start_time_ms))
```

Comment:
Counting CPU instructions as php-src https://github.com/php/php-src/tree/master/benchmark does might be better and much more comparable across runs/changes as much more stable.

Reply:
@mvorisek that is a great idea, I was not aware of this benchmark. I will try to implement something similar here